app/uk/gov/hmrc/economiccrimelevyregistration/config/AppConfig.scala

```diff
@@ -92,6 +92,6 @@ class AppConfig @Inject() (configuration: Configuration, servicesConfig: Service
 
   val privateBetaAccessCodes: Seq[String] = configuration.get[Seq[String]]("features.privateBeta.accessCodes")
   val privateBetaContactEmail: String     = configuration.get[String]("features.privateBeta.contactEmail")
-  val privateBetaEnabled: Boolean         = configuration.get[Boolean]("features.privateBeta.enabled")
+  val privateBetaEnabled: Boolean         = false // configuration.get[Boolean]("features.privateBeta.enabled")
```

Comment:
Please undo this

Reply:
Did you push any commit?

Reply:
Long story short. I turned off to test my other changes and gorgot to undo before the commit. Now done for sure